openmetadata-ui-core-components/src/main/resources/ui/src/components/MuiTabs/Tabs.stories.tsx

```diff
@@ -0,0 +1,341 @@
+/*
+ *  Copyright 2025 Collate.
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+import { Box, ThemeProvider, Typography } from "@mui/material";
+import type { Meta } from "@storybook/react";
+import React, { useEffect, useState } from "react";
+import { createMuiTheme } from "../../theme/createMuiTheme";
+import { Tabs, type TabsProps, type TabItem } from "./MuiTabs";
+
+interface CommonTabPanelPropsType {
+  children?: React.ReactNode;
+  index: string;
+  value: string;
+}
+
+function CommonTabPanel(props: CommonTabPanelPropsType) {
+  const { children, value, index, ...other } = props;
+
+  return (
+    <div
+      role="tabpanel"
+      hidden={value !== index}
+      id={`common-tabpanel-${index}`}
+      aria-labelledby={`common-tab-${index}`}
+      {...other}
+    >
+      {value === index && <Box sx={{ p: 3 }}>{children}</Box>}
+    </div>
+  );
+}
+
+type CustomTabsArgs = TabsProps;
+
+export const CustomTabs = (args: CustomTabsArgs) => {
```

Comment:
We should segregate every function, all types and constants to a separate file for better readability.